2-Authorization-I/1-call-graph/AppCreationScripts/sample.json

```diff
@@ -18,20 +19,19 @@
             "HomePage": "http://localhost:4200/",
             "ReplyUrls": "http://localhost:4200/",
             "Audience": "AzureADMyOrg",
+            "Sample": {
+                "SampleSubPath": "2-Authorization-I\\1-call-graph",
+                "ProjectDirectory": "\\SPA"
+            },
             "RequiredResourcesAccess": [
                 {
                     "Resource": "Microsoft Graph",
-                    "DelegatedPermissions": [
-                        "User.Read"
-                    ]
-                },
-                {
-                    "Resource": "Windows Azure Service Management API",
-                    "DelegatedPermissions": [
-                        "user_impersonation"
-                    ]
+                    "DelegatedPermissions": ["User.Read", "Contacts.Read"]
                 }
-            ]
+            ],        
+        "OptionalClaims": {
+          "IdTokenClaims": [ "acct" ]
```

Comment:
>acct [](http://example.com/codeflow?start=30&length=4)

Add a section in code/readme somewhere that talks about this optional claim (only applicable to apps that sign-in users). see this -> [https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2/blob/df985c70b25446bcccb5285f5ce19ca16dcb384a/4-WebApp-your-API/4-1-MyOrg/Client/Startup.cs#L114](https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2/blob/df985c70b25446bcccb5285f5ce19ca16dcb384a/4-WebApp-your-API/4-1-MyOrg/Client/Startup.cs#L114)